### PR TITLE
chore: Project update to support fabric-mod update (HL Fabric latest)

### LINKED ISSUE #132: Project update to support fabric-mod update (HL Fabric latest)

### PR DESCRIPTION
**Pull request by rolsonquadras**
Closes #132

Signed-off-by: Rolson Quadras <rolson.quadras@securekey.com>

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -6,18 +6,18 @@ module github.com/trustbloc/fabric-peer-ext
...
 	github.com/stretchr/testify v1.3.0
-	github.com/willf/bitset v1.1.9
-	go.uber.org/zap v1.9.1
+	github.com/willf/bitset v1.1.10
+	go.uber.org/zap v1.10.0
 
 )
 
-replace github.com/hyperledger/fabric => github.com/trustbloc/fabric-mod v0.0.0-20190522001819-bbf763a5c881
+replace github.com/hyperledger/fabric => github.com/rolsonquadras/fabric-mod v0.0.0-20190528172130-04c3e994144a
```
Comment (troyronda) [inline go.mod]: This should be reverted.

File: mod/peer/go.mod [hunk 2]
```diff
@@ -4,15 +4,16 @@
 
 module github.com/trustbloc/fabric-peer-ext/mod/peer
 
-replace github.com/hyperledger/fabric => github.com/trustbloc/fabric-mod v0.0.0-20190522001819-bbf763a5c881
+replace github.com/hyperledger/fabric => github.com/rolsonquadras/fabric-mod v0.0.0-20190528172130-04c3e994144a
```
Comment (troyronda) [inline mod/peer/go.mod]: This should be reverted.

Comment (rolsonquadras) [reply mod/peer/go.mod]: This is WIP. Not ready for review.

File: go.mod (hunk 1 above)
Comment (rolsonquadras) [reply go.mod]: This is WIP. Not ready for review.

File: pkg/config/config.go [hunk 3]
```diff
@@ -139,3 +144,81 @@ func GetOLCollPullTimeout() time.Duration {
 	}
 	return timeout
 }
+
+// GetPeerRootPath returns the peer file system root path
+func GetPeerRootPath() string {
```
Comment (troyronda) [inline pkg/config/config.go]: I assume there is a TODO to refactor how config is handled/injected?

Comment (rolsonquadras) [reply pkg/config/config.go]: Yeah, I'll add a TODO and create an issue.

**Comment by rolsonquadras** [comment]
verify-unitest

File: pkg/config/config.go (hunk 3 above)
Comment (troyronda) [reply pkg/config/config.go]: Cool - can you please link the issue?

Comment (rolsonquadras) [reply pkg/config/config.go]: Here is the issue - #137

**Comment by fqutishat** [comment]
verify-bddtest

**Comment by rolsonquadras** [comment]
verify-integration

**Pull request merged**